package.json

```diff
@@ -30,6 +30,7 @@
   "devDependencies": {
     "babel-loader": "^8.1.0",
     "chromedriver": "^83.0.1",
+    "chromium": "^3.0.2",
```

Comment:
are we still using puppeteer then when we have both chromium and chromedriver dependencies?

Reply:
Yes puppeteer is still used in the test started by "npm test"

Reply:
but puppeteers role is to get chromium and chromedriver versions that match and work, why do we both do that and hard code a version that happens to work right now? Also how are we to maintain these hard-coded versions?

Reply:
OK, removed puppeteer, karma is using the versions we selected. Now we have downloading scripts in package-json, the version of those can be maintained by npm. We have to maintain and match the versions of chromebrowser and chromedriver ourselves (set in .npmrc)

Reply:
alright I'm fine with that now that pupeteer is gone.